### DISCUSSION TITLE
Flux extension version roadmap

### DISCUSSION THREAD

**Discussion by danieltoth55** [Azure/AKS#3560] [Q&A] [answered]
Hey,

I was wondering if you have a timeline on releasing new versions of the Flux extension. The current version (1.7.0) uses Flux v0.39.0, but I would really need v0.40.0+.
As far as I know it is not possible to patch individual controllers when using the extension, so I'm kinda stuck on waiting for a new release of the extension.

Thanks!

**Comment by Jaykul** [comment]
Yes, and when will we get the 2.0.x GA versions, since they seem to support Workload Identity...

**Comment by philwelz** [comment, marked as answer]
You can find the versions [here](https://learn.microsoft.com/en-us/azure/azure-arc/kubernetes/extensions-release#flux-gitops). Its no roadmap but helps.
